### ISSUE TITLE
Length validation is broken for nil values

### ISSUE THREAD

**Issue by anthonator** [sticksnleaves/justify#1]
```
** (CaseClauseError) no case clause matching: :graphemes
     code: Services.Register.execute(command)
     stacktrace:
       (justify) lib/justify.ex:176: Justify.validate_length/3
       (primary_context) lib/primary_context/services/register.ex:26: PrimaryContext.Services.Register.validate_command/1
       (primary_context) lib/primary_context/services/register.ex:9: PrimaryContext.Services.Register.execute/1
       test/primary_context/lib/services/register_test.exs:30: (test)
```

**Comment by anthonator** [comment]
Ecto seems to handle length validation of `nil` and empty binary values by skipping them. The convention is to use `validate_required/3` instead.

**Comment by anthonator** [comment]
Fixed by #2
